changelog/unreleased/pull-5170

```diff
@@ -1,6 +1,22 @@
-BugFix: Prevent Windows VSS event log 8194
+Bugfix: Prevent Windows VSS event log 8194 from being created with `restic backup --use-fs-snapshot`
 
-Initialize VSS security to prevent VSS 8194 event logs occurring when running `restic backup` with `--use-fs-snapshot` flag in Windows.
+When running `restic backup` with `--use-fs-snapshot` flag in Windows with admin right, event logs like
+```
+Volume Shadow Copy Service error: Unexpected error querying for the IVssWriterCallback interface.  hr = 0x80070005, Access is denied.
+. This is often caused by incorrect security settings in either the writer or requestor process. 
+
+Operation:
+   Gathering Writer Data
+
+Context:
+   Writer Class Id: {e8132975-6f93-4464-a53e-1050253ae220}
+   Writer Name: System Writer
+   Writer Instance ID: {54b151ac-d27d-4628-9cb0-2bc40959f50f}
+```
+are created several times(the backup itself succeeds).
+To resolve this, initialize VSS security by calling `CoInitializeSecurity` following the approach from
+https://github.com/microsoft/Windows-classic-samples/blob/main/Samples/VShadowVolumeShadowCopy/cpp/vssclient.cpp#L52-L63 
```

Comment:
That reference is wrong, the PR uses the options from https://learn.microsoft.com/en-us/windows/win32/vss/security-considerations-for-requestors , and not from that sample. But either way, that's too technical for the changelog. If someone want the details, they can look at the pull request.